### PR TITLE
Move Getting Started contents to new doc site

### PR DESCRIPTION
**Pull request by htdvisser**
#### Summary

This Pull Request moves the contents of the Getting Started document to the new documentation site. It does **not** change the content, we'll do that in a later Pull Request.

#### Changes

- Move all Getting Started sections to the documentation site

image

#### Notes for Reviewers

I updated https://ttn.fyi/v3/getting-started to point to the latest tag's gettingstarted.md.

### CODE REVIEW

File: doc/content/gettingstarted/cli/_index.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+title: "Getting Started with the Command-line Interface"
```
Comment (adriansmares) [inline doc/content/gettingstarted/cli/_index.md]: Maybe this should be only `Command-line Interface` ? We are stuttering `Getting Started - Getting started with the Command-line Interface`

File: doc/content/gettingstarted/http/_index.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+title: "Getting Started with the HTTP API"
```
Comment (adriansmares) [inline doc/content/gettingstarted/http/_index.md]: Same as above.

File: doc/content/gettingstarted/mqtt/_index.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+title: "Getting Started with the MQTT API"
```
Comment (adriansmares) [inline doc/content/gettingstarted/mqtt/_index.md]: Same as above.

File: doc/content/gettingstarted/mqtt/subscribe.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    "dev_eui": "4200000000000000",
+    "join_eui": "4200000000000000",
+    "dev_addr": "01DA1F15"
+  },
+  "correlation_ids": [
+    "gs:conn:01D2CSNX7FJVKQPCVG612QF1TX",
+    "gs:uplink:01D2CT834K2YD17ZWZ6357HC0Z",
+    "ns:uplink:01D2CT834KNYD7BT2NHK5R1WVA",
+    "rpc:/ttn.lorawan.v3.GsNs/HandleUplink:01D2CT834KJ4AVSD1SJ637NAV6",
+    "as:up:01D2CT83AXQFQYQ35SR74CTWKH"
```
Comment (adriansmares) [inline doc/content/gettingstarted/mqtt/subscribe.md]: `as:up:01` renders a funny character in my browser:
```"as🆙01D2CV8HNGJ57G25BW0FCZNY07"```

File: doc/content/gettingstarted/mqtt/subscribe.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+    "dev_eui": "4200000000000000",
+    "join_eui": "4200000000000000",
+    "dev_addr": "01DA1F15"
+  },
+  "correlation_ids": [
+    "gs:conn:01D2CSNX7FJVKQPCVG612QF1TX",
+    "gs:uplink:01D2CV8HF62ME0D7MZWE38HHH8",
+    "ns:uplink:01D2CV8HF6FYJHKZ45YY1DB3MR",
+    "rpc:/ttn.lorawan.v3.GsNs/HandleUplink:01D2CV8HF6XR7ZFVK768PDG3J4",
+    "as:up:01D2CV8HNGJ57G25BW0FCZNY07"
```
Comment (adriansmares) [inline doc/content/gettingstarted/mqtt/subscribe.md]: Same as above

File: doc/content/gettingstarted/http/events.md [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+          "application_id": "app1"
+        },
+        "dev_eui": "4200000000000000",
+        "join_eui": "4200000000000000",
+        "dev_addr": "0063ECE2"
+      }
+    }
+  ],
+  "correlation_ids": [
+    "as:up:01D7KWB0VX1D7G3RKFN9HDA39Q",
```
Comment (adriansmares) [inline doc/content/gettingstarted/http/events.md]: Same as above

File: doc/content/gettingstarted/http/events.md [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
```
Comment (adriansmares) [inline doc/content/gettingstarted/http/events.md]: Same as above.

File: doc/content/gettingstarted/mqtt/subscribe.md (hunk 4 above)
Comment (htdvisser) [reply doc/content/gettingstarted/mqtt/subscribe.md]: Fixed

File: doc/content/gettingstarted/http/events.md (hunk 6 above)
Comment (htdvisser) [reply doc/content/gettingstarted/http/events.md]: Fixed

File: doc/content/gettingstarted/http/events.md (hunk 7 above)
Comment (htdvisser) [reply doc/content/gettingstarted/http/events.md]: Fixed

**Comment by htdvisser** [comment]
Changed the weight of sections. @adriansmares I don't think we can (or should want to) always keep that happy flow.

**Comment by htdvisser** [comment]
Oh Really? I thought we agreed from the start that we'd only use https://ttn.fyi/v3/getting-started

**Comment by johanstokking** [comment]
I see that url for the first time tbh

**Comment by htdvisser** [comment]
So what do you propose to put in that `gettingstarted.md` placeholder then, @johanstokking? We don't have a published docs site yet so we can't link to that yet. Maybe just to https://ttn.fyi/v3/getting-started ?

**Comment by johanstokking** [comment]
Sounds good!

**Comment by htdvisser** [comment]
@johanstokking @Sypheos @KrishnaIyer anything else? Otherwise I propose to merge this and improve the content in future pull requests.

**Pull request merged**